problem/0000-rfc-template.md

```diff
@@ -0,0 +1,71 @@
+- Contribution Name: Thrift Support
+- Implementation Owner: Peter Novotank <peter.novotnak@reddit.com>, Linkerd2 Authors
+- Start Date: 6/3/2020
+- Target Date: 6/15/2020
+- RFC PR: [linkerd/rfc#9](https://github.com/linkerd/rfc/pull/9)
+- Linkerd Issue: [linkerd/linkerd2#0000](https://github.com/linkerd/linkerd2/issues/0000)
+- Reviewers: (Who should review the code deliverable? ex.@olix0r)
+
+# Summary
+
+[summary]: #summary
+
+Linkerd2-proxy and control plane support for the Thrift protocol.
+
+# Problem Statement (Step 1)
+
+[problem-statement]: #problem-statement
+
+- Add protocol support to the linkerd2-proxy for at least TFramedTransport and TBinaryProtocol 
+- Support routing, retries and metrics for Thrift.
```

Comment:
For HTTP, retryability is configured through service profiles.  How should service profiles interact with thrift traffic? A service profile defines a list of routes which are typically matched based on HTTP method and path regexes.  This doesn't translate very well to thrift where there are no HTTP methods or paths.  Should we extend service profiles in some way to accommodate thrift traffic?  Perhaps by allowing routes to match on the thrift service name or thrift method name? Or should we use some different mechanism to signal thrift retryability?